common/reviews/api/rush-lib.api.ts

```diff
@@ -475,6 +495,7 @@ class VersionControl {
   public static getChangedFiles(prefix?: string, targetBranch?: string): string[];
   // (undocumented)
   public static getChangedFolders(targetBranch?: string): string[];
+  public static getUncommittedChanges(): string[];
```

Comment:
Fixed the new one. We can fix other APIs when doing major upgrade of Rush.

---
In reply to: [134069569](https://github.com/Microsoft/web-build-tools/pull/277#discussion_r134069569) [](ancestors = 134069569)